python/src/MaximumEntropyOrderStatisticsDistribution_doc.i.in

```diff
@@ -60,8 +105,8 @@ Draw a sample:
 
 Returns
 -------
-coll : sequence
-    The marginals."
+coll : of :class:`~openturns.Distribution`
```

Comment:
coll : sequence of :class:`~openturns.Distribution`

Reply:
fixed